multi-int-store/src/main/resources/application.yml

```diff
@@ -1 +1,12 @@
-callback-accept-version: 0.1.0-SNAPSHOT
\ No newline at end of file
+callback-accept-version: 0.1.0-SNAPSHOT
+logging:
+  level:
+    root: ERROR
+    org.springframework.web.filter.CommonsRequestLoggingFilter: DEBUG
+    com.connexta: DEBUG
+management:
+  endpoints:
+    web:
+      exposure:
+        include: "*"
+        exclude: "/actuator"
```

Comment:
✏️ Add documentation about how to use the actuator

Reply:
Added a new section to the README, "Debugging" and sub section for "Spring Actuator"